addons/website_sale_delivery/models/sale_order.py

```diff
@@ -72,3 +74,8 @@ def _cart_update(self, *args, **kwargs):
         """ Override to update carrier quotation if quantity changed """
         self._remove_delivery_line()
         return super()._cart_update(*args, **kwargs)
+
+    def _get_access_point_address(self):
+        if self.access_point_address:
+            return json.loads(self.access_point_address)
+        return False
```

Comment:
This function is a bit useless since you only call it once and it's not doing a lot except a json.loads. It could be in the controller directly